apps/src/applab/applab.js

```diff
@@ -337,6 +344,26 @@ function onDebugInputKeyDown(e) {
       outputApplabConsole('< (not running)');
     }
   }
+  if (e.keyCode == KeyCodes.UP) {
```

Comment:
Our style convention is to prefer `===` whenever possible.

It looks like we didn't follow our own rule in this function before and I can't find a reference to it in our style guide.  Oops!  I'll follow up on that here.